frontend/src/routes/index.svelte

```diff
@@ -160,7 +160,9 @@
 
              --borderRadius: var(--rounded-btn, .5rem);
              --background: {DT[`[data-theme=${$chosenTheme}]`]['base-100']};
-             --border: 1px solid {DT[`[data-theme=${$chosenTheme}]`]['primary']};
+             --border: 1px solid {DT[`[data-theme=${$chosenTheme}]`][
+      `${viewPortWidth <= 640 ? '' : 'primary'}`
```

Comment:
Could we do this with the Navbar MediaQuery idea instead, so it happens dynamically?

Also, both the multi-selects should probably be a component - Would look so nice if index didn't have this crazy CSS madness